src/pages/Account/accountUtils.ts

```diff
@@ -18,6 +19,22 @@ const expressions = {
   type: new RegExp('^(?<type>[A-Za-z]+)'),
 }
 
+export const mapBridgeIdToDisplayName = (bridgeId: BridgeList) => {
+  switch (bridgeId) {
+    case 'arbitrum:mainnet':
+    case 'arbitrum:testnet':
+      return 'Arbitrum'
+    case 'connext':
+      return 'Connext'
+    case 'omnibridge:eth-xdai':
+      return 'Omnibridge'
+    case 'socket':
+      return 'Socket'
+    case 'xdai':
+      return 'xDai'
+  }
+}
```

Comment:
In this case is better to create a dictionary and map key to value :)

Reply:
Also maybe we should have a different name for `Arbitrum mainnet` and `Arbitrum testnet` as it may confuse users.